oem/firstvoices/ios/FirstVoices/LexicalModelRepository.swift

```diff
@@ -0,0 +1,83 @@
+/*
+ * LexicalModelRepository.swift
+ * FirstVoices app
+ *
+ * License: MIT
+ *
+ * Copyright © 2022 FirstVoices.
+ *
+ * Created by Shawn Schantz on 2022-01-27.
+ *
+ * Class to call the Keyman API to query about lexical models for a given language.
+ *
+ */
+
+import UIKit
+import KeymanEngine
+
+class LexicalModelRepository {
+
+  private static let keymanLexicalModelApiUrl = "https://api.keyman.com/model?q=bcp47:"
```

Comment:
I wonder if some of this should use existing methods in

https://github.com/keymanapp/keyman/blob/master/ios/engine/KMEI/KeymanEngine/Classes/Queries/Queries%2BLexicalModel.swift#L45